<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled" >
   <bool>true</bool>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout_4" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
       <property name="tabShape" >
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>With text</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <layout class="QVBoxLayout" name="verticalLayout_5" >
           <item>
            <widget class="QListWidget" name="withTextListWidget" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="editTriggers" >
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation" >
              <bool>true</bool>
             </property>
             <property name="showDropIndicator" stdset="0" >
              <bool>false</bool>
             </property>
             <property name="selectionMode" >
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="iconSize" >
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="movement" >
              <enum>QListView::Static</enum>
             </property>
             <property name="flow" >
              <enum>QListView::LeftToRight</enum>
             </property>
             <property name="isWrapping" stdset="0" >
              <bool>true</bool>
             </property>
             <property name="resizeMode" >
              <enum>QListView::Adjust</enum>
             </property>
             <property name="layoutMode" >
              <enum>QListView::Batched</enum>
             </property>
             <property name="spacing" >
              <number>10</number>
             </property>
             <property name="gridSize" >
              <size>
               <width>170</width>
               <height>200</height>
              </size>
             </property>
             <property name="viewMode" >
              <enum>QListView::IconMode</enum>
             </property>
             <property name="uniformItemSizes" >
              <bool>true</bool>
             </property>
             <property name="batchSize" >
              <number>100</number>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" >
             <item>
              <spacer name="horizontalSpacer" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="prevBtn" >
               <property name="text" >
                <string/>
               </property>
               <property name="icon" >
                <iconset resource="../gui.qrc" >
                 <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox" >
               <item>
                <property name="text" >
                 <string>Page 1</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="nextBtn" >
               <property name="text" >
                <string/>
               </property>
               <property name="icon" >
                <iconset resource="../gui.qrc" >
                 <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <attribute name="title" >
         <string>Without text</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6" >
         <item>
          <widget class="QListWidget" name="withoutTextListWidget" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutMode" >
            <enum>QListView::Batched</enum>
           </property>
           <property name="batchSize" >
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QDockWidget" name="dockWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>550</width>
         <height>524287</height>
        </size>
       </property>
       <widget class="QWidget" name="dockWidgetContents" >
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" >
          <layout class="QVBoxLayout" name="verticalLayout_7" >
           <item>
            <spacer name="verticalSpacer_3" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="verticalSpacer_4" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGraphicsView" name="graphicsView" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>350</width>
               <height>512</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>512</width>
               <height>512</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3" >
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" >
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2" >
                 <property name="sizeConstraint" >
                  <enum>QLayout::SetNoConstraint</enum>
                 </property>
                 <item>
                  <widget class="QLabel" name="label" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text" >
                    <string>Annnotations</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer" >
                   <property name="orientation" >
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="textEdit" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout" >
                 <item>
                  <widget class="QPushButton" name="textOnlyButton" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text" >
                    <string>View text only</string>
                   </property>
                   <property name="checkable" >
                    <bool>true</bool>
                   </property>
                   <property name="autoExclusive" >
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="fullImageButton" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text" >
                    <string>View full image</string>
                   </property>
                   <property name="checkable" >
                    <bool>true</bool>
                   </property>
                   <property name="autoExclusive" >
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="textBoxesButton" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text" >
                    <string>View text boxes</string>
                   </property>
                   <property name="checkable" >
                    <bool>true</bool>
                   </property>
                   <property name="autoExclusive" >
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="textMaskButton" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text" >
                    <string>View text mask</string>
                   </property>
                   <property name="checkable" >
                    <bool>true</bool>
                   </property>
                   <property name="autoExclusive" >
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_2" >
                   <property name="orientation" >
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" >
               <item>
                <spacer name="horizontalSpacer_3" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text" >
                  <string>Validate</string>
                 </property>
                 <property name="icon" >
                  <iconset resource="../gui.qrc" >
                   <normaloff>:/icons/ok.png</normaloff>:/icons/ok.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <spacer name="verticalSpacer_5" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionScan_directory" />
    <addaction name="separator" />
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionFull_Image" />
    <addaction name="actionText_Boxes" />
    <addaction name="separator" />
    <addaction name="action_Text_Only" />
    <addaction name="actionText_Mask" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_View" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionScan_directory" >
   <property name="icon" >
    <iconset resource="../gui.qrc" >
     <normaloff>:/icons/photos.png</normaloff>:/icons/photos.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Scan directory...</string>
   </property>
  </action>
  <action name="action_Text_Only" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Text only</string>
   </property>
  </action>
  <action name="actionFull_Image" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Full image</string>
   </property>
  </action>
  <action name="actionText_Boxes" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Text boxes</string>
   </property>
  </action>
  <action name="actionText_Mask" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Text mask</string>
   </property>
  </action>
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../gui.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>353</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFull_Image</sender>
   <signal>activated()</signal>
   <receiver>fullImageButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>634</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Text_Only</sender>
   <signal>activated()</signal>
   <receiver>textOnlyButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>634</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionText_Boxes</sender>
   <signal>activated()</signal>
   <receiver>textBoxesButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>634</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionText_Mask</sender>
   <signal>activated()</signal>
   <receiver>textMaskButton</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>634</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
